lib/clusterbuster/workloads/fio.workload

```diff
@@ -265,8 +264,8 @@ function fio_report_options() {
 EOF
 }
 
-function fio_namespace_policy() {
-    echo privileged
+function fio_requires_drop_cache() {
+    :
```

Comment:
I think you should add it on separate PR

Reply:
See above.